### DISCUSSION TITLE
Long playlist names are cut off by overlay

### DISCUSSION THREAD

**Discussion by plaguemask** [ppy/osu#16306] [Q&A]
![osu!_2022-01-02_20-43-49](https://user-images.githubusercontent.com/66150801/147896651-aa455766-4168-4c1e-bff2-144954441b3d.png)

I'd expect a convenient way to read the full name, like a tooltip.

**Comment by peppy** [comment]
Known issue, until we figure out how to truncate this display properly (see trail at https://github.com/ppy/osu/discussions/15450).
